integration/auth/user_mgt_test.go

```diff
@@ -447,10 +449,12 @@ func TestCreateUserMFA(t *testing.T) {
 	defer deleteUser(user.UID)
 	var factor []*auth.MultiFactorInfo = []*auth.MultiFactorInfo{
 		{
-			UID:                 user.MultiFactor.EnrolledFactors[0].UID,
-			DisplayName:         "Spouse's phone number",
-			FactorID:            "phone",
-			PhoneNumber:         "+11234567890",
```

Comment:
Since the `PhoneNumber` is deprecated and not dropped, let's keep `PhoneNumber` in tests for now to check backward compatibility